docs/cli/scan-in-osh.md

```diff
@@ -0,0 +1,29 @@
+---
+title: scan-in-osh
+date: 2024-05-22
+sidebar_position: 41
+
+---
+
+# `packit scan-in-osh`
+
+Builds an SRPM file and performs a scan in [OpenScanHub])(https://openscanhub.fedoraproject.org/).
+
+## Help
+
+    Usage: packit scan-in-osh [OPTIONS] [PATH_OR_URL]
+
+      Perform a scan through OpenScanHub. You need a valid kerberos ticket and set
```

Comment:
```suggestion
      Perform a scan through OpenScanHub. You need a valid Kerberos ticket and set
```

Reply:
This needs to be fixed in the CLI to preserve this..;) I'll take care of this and the other suggestion.. Thanks!

Reply:
Here it is: https://github.com/packit/packit/pull/2319